pypit/artrace.py

```diff
@@ -389,14 +389,18 @@ def trace_objbg_image(slf, det, sciframe, slitn, objreg, bgreg, trim=2, triml=No
     for o in range(nobj):
         wll = np.where(bgreg[0][1:, o] > bgreg[0][:-1, o])[0]
         wlr = np.where(bgreg[0][1:, o] < bgreg[0][:-1, o])[0]
+        if len(wll) < len(wlr): #< len(wlr): # JXP kludge
+            wll = np.concatenate([np.array([0]).astype(int), wll])
         # Background regions to the left of object
-        for ii in range(wll.size):
```

Comment:
I don't think this change is needed, but perhaps we should assert that `wll` and `wlr` are the same size before the for loop?

Reply:
I did find a case where wlr had zero lengths and
wll was not.  Am not sure how the code should
proceed in that case aside from skipping the for loop.